python/cml/torch/memory.py

```
@@ -0,0 +1,48 @@
+    Not thread-safe: do not share one instance across threads while another
+    thread may call close() or rely on garbage-collection teardown.
+    Prefer explicit close() or a with-block in multi-threaded hosts.
+    """
+
+    def __init__(self, size: int):
+        self._mgr = lib.torch_memory_create(size)
+        if self._mgr == ffi.NULL:
+            raise MemoryError(f"Failed to allocate {size}-byte arena")
+
+    @classmethod
+    def from_buffer(cls, buffer, size: int) -> "MemoryManager":
```

Ruff (0.15.17)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)